app/views/layouts/listnav/_ems_physical_infra.html.haml

```diff
@@ -0,0 +1,60 @@
+- if @record.try(:name)
+  - hosts_title    = title_for_hosts
+  - clusters_title = title_for_clusters
+  #accordion.panel-group
+    = miq_accordion_panel(truncate(@record.name, :length => truncate_length), true, "icon") do
+      = render_quadicon(@record, :mode => :icon, :size => 72, :typ => :listnav)
+
+    = miq_accordion_panel(_("Properties"), false, "ems_prop") do
+      %ul.nav.nav-pills.nav-stacked
+        %li
+          = link_to_with_icon(_('Summary'), polymorphic_path(@record, :display => 'main'), :title => _("Show Summary"))
+        = li_link(:if => (@record.has_events? || @record.has_events?(:policy_events)),
+            :text       => _('Timelines'),
+            :record     => @record,
+            :display    => 'timeline',
+            :title      => _("Show Timelines"))
+
+    = miq_accordion_panel(_("Relationships"), false, "ems_rel") do
+      %ul.nav.nav-pills.nav-stacked
+        = li_link(:if => !(@record.number_of(:ems_folders) == 0 || @record.ems_folder_root.nil?),
+          :text       => _("%{hosts} & %{clusters}") % {:hosts => hosts_title, :clusters => clusters_title},
```

Comment:
Do we have to do the openstack thing here too?

Please correct me if I'm wrong but should this be "Physical servers" instead of "VMs & Templates" or "Hosts & Clusters"?